e2e/bin/update

```diff
@@ -1,17 +1,17 @@
 #!/bin/bash
 
+set -e
+
 if [ $# -ne 2 ]; then
-    echo "./upload.sh <source> <destination>"
+    echo "$0 <source> <destination>" 2>&1
     exit 1
 fi
 
-set -e
+. `dirname $0`/nodes-opts
```

Comment:
Some shell-scripting nitpicks:

```suggestion
. $(dirname ${BASH_SOURCE[0]})/nodes-opts
```

We don't shellcheck our scripts, but if we did it would recommend command substitution over backticks. It doesn't _really_ matter here but it's nice to be consistent.

Usually you should prefer `${BASH_SOURCE[0]}` over `$0` because `$0` can be set by a caller.

Reply:
ah, I should have run shellcheck on it. Will do